### ISSUE TITLE
iOS requires info.plist key NSCalendarsUsageDescription

### ISSUE THREAD

**Issue by JamieWhite** [ProcedureKit/ProcedureKit#460]
Now then,

I've had a iOS 10 build rejected by Apple because it was missing a few `info.plist` keys. One of them is `NSCalendarsUsageDescription`. I'm not using the calendar anywhere in my application so I `grep`'d 
all my dependency source code and `ProcedureKit` / `Operations` comes up. I've filled this for now but it's something to consider going forward with your refactor. If you decide to keep the contacts stuff in the main `.framework` then it would be good to include this in the `README`.

Here is the full error text I got:

```
This app attempts to access privacy-sensitive data without a usage description. The app's Info.plist must contain an NSContactsUsageDescription key with a string value explaining to the user how the app uses this data.

This app attempts to access privacy-sensitive data without a usage description. The app's Info.plist must contain an NSCalendarsUsageDescription key with a string value explaining to the user how the app uses this data.
```

I've included the Contacts one in there because you're also using contacts, although I was using my own contacts stuff so that was fine.

**Comment by danthorpe** [comment]
Hi - this is fixed for integration via CocoaPods in v3.3.1

However - for Carthage (which I think you use, it will not work)

At the moment - your best option is to fork - remove the CalendarCapability.swift and then you can stick with Carthage. Alternatively - integrate with CocoaPods?

Lastly - there is a wiki page which covers submitting for iOS 10 - see the Home page of the wiki.

**Comment by JamieWhite** [comment]
It's no big deal to me for now to be honest. I want to stay away from changing too much in my forks. My current `ProcedureKit` fork is just deleting the main `.xcodeproj` so Carthage builds the `Extension_compatible` project. That's much easier than what it was previously.

and I'm sorry, I didn't spot the iOS 10 wiki page. Most of the time the README is all I read 😄
